test_agasc1p7.py

```diff
@@ -0,0 +1,47 @@
+
+import numpy as np
+import tables
+
+h5 = tables.openFile('agasc1p7.h5', mode='r')
+h5inp = tables.openFile('/proj/sot/ska/data/agasc/agasc1p7.h5', mode='r')
+h51p6 = tables.openFile('/proj/sot/ska/data/agasc/agasc1p6.h5', mode='r')
+
+assert np.all(h5.root.data.colnames == h51p6.root.data.colnames)
+assert np.all(h5.root.data.colnames == h5inp.root.data.colnames)
+
+colnames = h5.root.data.colnames
+
+# stars with no changes
+ok = h5.root.data.col('RSV3') == 0
+
+agasc_ids = h5.root.data.col('AGASC_ID')
+agasc_ids2 = h5inp.root.data.col('AGASC_ID')
+agasc_ids3 = h51p6.root.data.col('AGASC_ID')
+
+idx = np.arange(len(agasc_ids))
+
+id_map2 = dict(zip(agasc_ids2, idx))
+id_map3 = dict(zip(agasc_ids3, idx))
+
+h5inp_id_map = [id_map2[agasc_id] for agasc_id in agasc_ids]
```

Comment:
Why is this ID map required?  The files should have exactly the same number of rows, in exactly the same order, right?

Reply:
For some reason they don't have the same order. I'll double check this but without the ID map I was getting Assertion Errors.